cypress/integration/login.spec.js

```diff
@@ -1,94 +1,83 @@
 import login from "../pages/loginModal.json";
 import headerLocators from "../pages/header.json";
 import organizationLocators from "../pages/organization.json";
+import Login from "../support/classes/login/index.js";
+
+const login1 = new Login();
 
 describe("Login", () => {
+  beforeEach(() => {
+    login1.setupTests();
+  });
+
   it("Attempt to login with empty fields", () => {
-    cy.visit("/login");
-    cy.get(login.loginButton).click();
+    login1.loginWithEmptyCredentials();
 
-    cy.get(login.loginModal)
-      .should("contain.text", "The email field must be a valid email")
-      .and("contain.text", "The password field is required");
-    cy.url().should("eq", "https://cypress.vivifyscrum-stage.com/login");
+    login1.validateUnsuccessfullLoginEmptyFields(
+      "The email field must be a valid email",
```

Comment:
It would make more sense if this was an object. For example:

```suggestion
       login1.validateUnsuccessfullLoginEmptyFields({
           emailRequiredMessage: "The email field must be a valid email",
           passwordRequiredMessage: "The password field is required",
           url: "https://cypress.vivifyscrum-stage.com/login"
       });
```